### PR TITLE
Relicense: GPL -> BSD

### PR DESCRIPTION
**Pull request by oliver-sanders**
We are trying to relicense this repository from GPL (V3) to BSD (three clause) to allow it to be used for a wider range of purposes. In particular we would like to open this code to potential use in GitHub Linguist which would allow Cylc language detection and syntax highlighting to be added to GitHub. Note, this relicensing applies only to this repository (cylc-textmate-grammar), and not any of the core repositories.

I have copied across the CLA from the [main repository](https://github.com/cylc/cylc-flow/blob/master/CONTRIBUTING.md) (the one you've previously agreed to) unchanged. By re-agreeing to the CLA you agree to:

* The terms of the CLA
* The new license

All code authors, if agreed to the changes, please raise a PR to this branch adding your name to the CLA and copying any relevant lines from the `.mailmap` file from the on in [cylc-flow](https://github.com/cylc/cylc-flow/blob/master/.mailmap). (make sure your name in the contributing section exactly matches that in the mailmap):

* [x] @oliver-sanders 
* [x] @MetRonnie 
* [x] @wxtim 
* [x] @kinow 

Additionally, @markgrahamdawson, as you merged a commit, your name shows in the project history, it would make it easier if you could sign too:

* [x] @markgrahamdawson 

**Check List**

- [x] I have read `CONTRIBUTING.md` and added my name as a Code Contributor.
- [x] The "check contributor list" test will pass once all names are in the changelog and the mailmap is complete.

### CODE REVIEW

**Comment by kinow** [comment]
Incoming PR in a few minutes…

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+(c) The contribution was provided directly to me by some other person
+    who certified (a) or (b) and I have not modified it.
+
+(d) I understand and agree that this project and the contribution
+    are public and that a record of the contribution (including my
+    name and email address) is retained for the full term of
+    the copyright and may be redistributed consistent with this project
+    or the licence(s) involved.
+
+(e) I, or my employer, grant to NIWA and all recipients of
```
Comment (MetRonnie) [inline CONTRIBUTING.md:63]: Is this just standard spiel for all repos in the Cylc organisation, should it include the Met Office as well?

Comment (oliver-sanders) [reply CONTRIBUTING.md:63]: This is the CLA that was agreed at the time, tricky to change now.

**Comment by MetRonnie** [comment]
https://github.com/oliver-sanders/cylc-textmate-grammar/pull/2

**Review by MetRonnie — APPROVED** [review]

**Pull request merged**